lib/model/folder_sendrecv.go

```diff
@@ -17,6 +16,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
Error context traces - nice concept :)

There's quite a few places where it could be used as well, where we currently use a something custom like `fmt.Errorf("%s: %s", context, err)` - but that's unrelated to this PR (maybe it's the spark for it's usage elsewhere :) ).

Reply:
It is, and I've wanted to introduce it. I took the step to do so here because without it we would just get a `Lstat: foo: permission denied` or something which would be a lot of tricky debugging to figure out what the fuck is going on, while seeing the `copy owner from parent: Lstat: foo: ...` is a much better place to start from when figuring out why the stat (or whatever) fails.